packages/vc-handler-ld-local/src/__tests__/issue-verify-flow.test.ts

```diff
@@ -0,0 +1,203 @@
+            'did:lto': new LtoDidProvider({
+              defaultKms: 'local',
+              connectionMode: IDidConnectionMode.NODE,
+              network: 'T',
+              sponsorPrivateKeyBase58: '5gqCU5NbwU4gc62be39LXDDALKj8opj1KZszx7ULJc2k33kk52prn8D1H2pPPwm6QVKvkuo72YJSoUhzzmAFmDH8',
+            }),
+          },
+          store: new MemoryDIDStore(),
+          defaultProvider: 'did:key',
+        }),
+        new DIDResolverPlugin({
+          resolver: new Resolver({
+            ...getDidKeyResolver(),
+            ...getUniResolver('lto', { resolveUrl: 'https://lto-mock/1.0/identifiers' }),
+            ...getUniResolver('factom', { resolveUrl: 'https://factom-mock/1.0/identifiers' }),
+          }),
+        }),
+        new CredentialIssuer(),
+        new CredentialHandlerLDLocal({
+          contextMaps: [LdDefaultContexts, customContext],
+          suites: [new SphereonEd25519Signature2018(), new SphereonEd25519Signature2020()],
+          bindingOverrides: new Map([
+            // Bindings to test overrides of credential-ld plugin methods
+            ['createVerifiableCredentialLD', MethodNames.createVerifiableCredentialLDLocal],
+            ['createVerifiablePresentationLD', MethodNames.createVerifiablePresentationLDLocal],
+            // We test the verify methods by using the LDLocal versions directly in the tests
+          ]),
+        }),
+      ],
+    })
+    didKeyIdentifier = await agent.didManagerCreate()
+    didLtoIdentifier = await agent.didManagerImport({
+      provider: 'did:lto',
+      did: LTO_DID,
+      controllerKeyId: `${LTO_DID}#sign`,
+      keys: [
+        {
+          privateKeyHex:
+            '078c0f0eaa6510fab9f4f2cf8657b32811c53d7d98869fd0d5bd08a7ba34376b8adfdd44784dea407e088ff2437d5e2123e685a26dca91efceb7a9f4dfd81848',
+          publicKeyHex: '8adfdd44784dea407e088ff2437d5e2123e685a26dca91efceb7a9f4dfd81848',
+          kms: 'local',
+          kid: `${LTO_DID}#sign`,
+          type: 'Ed25519',
+        },
+      ],
+    })
+  })
+
+  it('works with Ed25519Signature2018', async () => {
+    nock('https://lto-mock/1.0/identifiers')
+      .get(`/${LTO_DID}`)
+      .times(3)
+      .reply(200, {
+        ...ltoDIDResolutionResult,
+      })
+    const credentialPayload: CredentialPayload = {
+      issuer: didKeyIdentifier.did,
+      type: ['VerifiableCredential', 'AlumniCredential'],
+      '@context': ['https://www.w3.org/2018/credentials/v1', 'https://www.w3.org/2018/credentials/examples/v1'],
+      credentialSubject: {
+        id: didLtoIdentifier.did,
+        alumniOf: {
+          id: 'did:example:c276e12ec21ebfeb1f712ebc6f1',
+          name: 'Example University',
+        },
+      },
+    }
+    const verifiableCredential = await agent.createVerifiableCredential({
+      credential: credentialPayload,
+      proofFormat: 'lds',
+    })
+
+    expect(verifiableCredential).toBeDefined()
+    // console.log(verifiableCredential)
```

Comment:
Don't forget to remove commented code.

Reply:
Mwah, I totally agree that you don't want to have full code structures in comments. Having a debug line, where it makes sense, so it can be reused by someone else later is not a biggy IMO